components/place_search.js

```diff
@@ -113,8 +113,7 @@ export class PlaceSearch extends Component {
               type="text"
               placeholder={this.props.t("map.search-location-prompt")}
               className={this.props.classes.searchInputField}
-              onKeyDown={this.onKeyPress}
-              onKeyUp={this.onKeyDown}
+              onKeyDown={() => this.onKeyPress()}
```

Comment:
Why not just give it this.onKeyPress?

Reply:
`onKeyUp` wasn't doing anything, I just removed it

Reply:
I meant why do the ()=>

Reply:
Hmm I think I did that original stuff? Had to do with whether the last char you typed was used if you hit the button instead of pressing Enter?

Reply:
I have found that it executed all code on render that is defined without the lambda